packages/mobile/src/web3/providers.ts

```diff
@@ -0,0 +1,55 @@
+import * as net from 'react-native-tcp'
```

Comment:
No new code here, just moved these functions from `web3/contracts` for better organization.